scripts/tutorials/gatk/tutorial_8017.wdl

```diff
@@ -0,0 +1,356 @@
+# Tutorial_8017.wdl #######################################################
+#
+# This WDL script recapitulates the main workflow outlined in Tutorial#8017 at
+# <https://software.broadinstitute.org/gatk/documentation/article?id=8017>.
+# The tutorial provides example input data including a mini-reference
+# that finishes processing in a few minutes.
```

Comment:
"mini-reference that finishes processing in a few minutes"

The reference doesn't *do* anything -- what you're saying is that the script execution is expected to take a short time.